davidgljay/nametag
# Typing prompt

## What does this PR do?
Adds a prompt showing when another user in a room is typing.

## How do I test this PR?
1. Enter a room
2. In an incognito window, enter the same room as a different user.
3. Start typing in one window. A typing prompt with the user's icon should appear in the other window.
4. Wait 3 seconds, the typing prompt should disappear.
5. Type continuously in the incognito window, the typing prompt should stay present in the other window.
6. Submit a message in the incognito window. The typing prompt should disappear.


## Notes
 @chromakode, would love your thoughts on how this is implemented before I merge if you have time. It should meet the following conditions.
